pinc/Project.inc

```diff
@@ -1076,6 +1079,15 @@ class Project
         return $PPVer;
     }
 
+    private function _get_round_type(): string
+    {
+        $round = get_Round_for_project_state($this->state);
+        if (!$round) {
+            return "none";
+        }
+        return is_formatting_round($round) ? "formatting" : "proofreading";
```

Comment:
It shouldn't, because of this check above it:
```php
        $round = get_Round_for_project_state($this->state);
        if (!$round) {
            return "none";
        }
```
PP, SR, and others are not Rounds so they will be `none`.

Reply:
Indeed, I forgot that we keep Rounds/Stages/Activities separately. Thanks for the correction!